homeassistant/components/comfoconnect/fan.py

```diff
@@ -9,21 +10,25 @@
     SENSOR_FAN_SPEED_MODE,
 )
 
-from homeassistant.components.fan import (
-    SPEED_HIGH,
-    SPEED_LOW,
-    SPEED_MEDIUM,
-    SPEED_OFF,
-    SUPPORT_SET_SPEED,
-    FanEntity,
-)
+from homeassistant.components.fan import SUPPORT_SET_SPEED, FanEntity
 from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.util.percentage import (
+    percentage_to_ranged_value,
+    ranged_value_to_percentage,
+)
 
 from . import DOMAIN, SIGNAL_COMFOCONNECT_UPDATE_RECEIVED, ComfoConnectBridge
 
 _LOGGER = logging.getLogger(__name__)
 
-SPEED_MAPPING = {0: SPEED_OFF, 1: SPEED_LOW, 2: SPEED_MEDIUM, 3: SPEED_HIGH}
+CMD_MAPPING = {
+    0: CMD_FAN_MODE_AWAY,
+    1: CMD_FAN_MODE_LOW,
+    2: CMD_FAN_MODE_MEDIUM,
+    3: CMD_FAN_MODE_HIGH,
+}
+
+SPEED_RANGE = (1, 3)  # off (CMD_FAN_MODE_AWAY) is not included
```

Comment:
We might want to include AWAY. It's not really off in this setting, but just the lowest setting. There is no way to turn of the ventilation unit, to avoid moist in the tubes.

Reply:
Although, this might also be correct. I'm not sure how to model the fan since it's not possible to turn it off. I initially used Away mode for off, since it's the lowest setting you can use (and it's really the equivalent for off, since on the official remote, the lowest setting is LOW.)

Reply:
I think what you did to map `away` to `off` makes sense in this case.

Since in this case we aren't treating `away` as a speed and rather `off`, its not included in the `speeds` since its mapped to `off`

Reply:
I updated the comment to reflect that.

Reply:
LGTM I think the average user would probably say "I turn off the air exchanger when I'm away"